### ISSUE TITLE
Rewrite

### ISSUE THREAD

**Issue by schorschii** [schorschii/RemotePointer-Server#4]
A rewrite in Python seems to be necessary.

- Java .jar applications require a Java runtime installed, a single binary would be more user-friendly (e.g. via PyInstaller)
- Java tray icon is buggy on Linux (transparency)
- Java applications cannot be displayed in fullscreen mode on macOS (bad for the pointer feature)
- it would be easier for users to grant the necessary permissions on macOS settings since with Java applications, you need to give them to "Java" in general and not to the specific application, which is not intuitive

**Comment by martin-braun** [comment]
Hi, I totally agree for not writing the server in Java, but notice Python is very slow. I think it would be much better to program the server in a language that compiles to native code, such as C++, Rust or Go. If you really want to pick Python, maybe Mojo would be a great outlook, but they are far from production ready.

**Comment by schorschii** [comment]
I don't think that speed really matters in this application. I'm not implementing time critical algorithms here. Btw. one experience of mine: the result of real-time RTP packet/audio decoding in Python is not that bad. I tried that myself in [Jabber4Linux](https://github.com/schorschii/Jabber4Linux).
